src/app/shared/conditional-show/conditional-show-view.directive.ts

```diff
@@ -0,0 +1,35 @@
+import { AfterViewInit, Directive, ElementRef, Input } from '@angular/core';
+import { CaseField } from '../domain/definition/case-field.model';
+import { ShowCondition } from './conditional-show.model';
+
+@Directive({selector: '[ccdConditionalShowView]'})
```

Comment:
why a new directive? 
have you tried using  ccdConditionalShow without passing the formGroup property? e.g.
<div ccdConditionalShowView ccdLabelSubstitutor [caseField]="field" [eventFields]="caseFields">

Reply:
I started with that option but got into trouble -which I don't remember clearly now- so I decided to remove some logic to have a simple new directive

Reply:
Would be nice to reuse the existing directive that @mario-paniccia suggested if it is not too hard to do?